packages/apps/plugins/plugin-attention/package.json

```diff
@@ -9,15 +9,17 @@
   "exports": {
     ".": {
       "browser": "./dist/lib/browser/index.mjs",
-      "import": "./dist/lib/browser/index.mjs",
-      "require": "./dist/lib/node/index.cjs",
-      "node": "./dist/lib/node/index.cjs"
+      "node": {
+        "default": "./dist/lib/node/index.cjs"
+      },
+      "types": "./dist/types/src/index.d.ts"
     },
     "./meta": {
       "browser": "./dist/lib/browser/meta.mjs",
-      "import": "./dist/lib/browser/meta.mjs",
-      "require": "./dist/lib/node/meta.cjs",
-      "node": "./dist/lib/node/meta.cjs"
+      "node": {
+        "default": "./dist/lib/node/meta.cjs"
```

Comment:
The idea here is to eventually have separate "import" and "require" exports for ESM and CJS respectively